capstone-frontend/src/components/TestApi.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react'
+
+export class TestApi extends Component {
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      testData: null,
+    };
+  }
+
+  componentDidMount() {
+
+    console.log("Mounted");
+
+    // fetch("https://8080-bbaec244-5a54-4467-aed6-91c386e88c1a.ws-us02.gitpod.io/testData", { credentials: 'include' })
+    //   .then(response => response.json())
+    //   .then(testData => this.setState({ testData }));
+
+    // fetch("https://8080-bbaec244-5a54-4467-aed6-91c386e88c1a.ws-us02.gitpod.io/testData", { credentials: 'include' })
+    //   .then(response => response.json())
+    //   .then(testData => console.log(testData));
+
+
+    fetch("/api/testData")
+      .then(response => response.json())
+      .then(testData => this.setState({ testData }));
+
+    fetch("/api/testData")
+      .then(response => response.json())
+      .then(testData => {
+        console.log(testData);
+      });
+  }
+
+
+  render() {
+    return (
+      <div>
+        <h1>Testing Api</h1>
+        <p>Fetch Data: {this.state.testData}</p>
+      </div>
+    )
+  }
+}
+
+export default TestApi
```

Comment:
`TestApi` is both a named export and the default export for this module. You should probably discuss with your teammates whether you want to use default exports, then remove one or the other here.

Reply:
Hi Geoff, I don't really know React much. Is there an alternative to default exports and how does it all work?